File: src/main/kotlin/com/example/chinaApp/dao/service/AnswerDaoService.java
package com.example.chinaApp.dao.service;

import com.example.chinaApp.dao.repository.AnswerRepository;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Service;

@Service
@Slf4j
@RequiredArgsConstructor
public class AnswerDaoService {

    private final AnswerRepository answerRepository;
}

File: src/main/kotlin/com/example/chinaApp/dao/service/AnswerDaoService.kt
package com.example.chinaApp.dao.service

import com.example.chinaApp.dao.entity.AnswerEntity
import com.example.chinaApp.dao.repository.AnswerRepository
import org.springframework.stereotype.Service
import java.util.*

@Service
class AnswerDaoService (
    private val answerRepository: AnswerRepository
) {

    fun findAnswersByQuestionId(questionId: UUID): List<AnswerEntity> {
        return answerRepository.findByQuestionId(questionId)
    }
}
